reference/channels.v3.yml

```diff
@@ -452,11 +452,16 @@ paths:
                 1.product_id: error.path.missing
       tags:
         - Channel Listings
+
+      description: >-
+        Updates a listing for provided channel
+
       description: |-
         Currently only the following properties can be updated on channel listings:
 
         * state
         * variants.state
```

Comment:
Need to remove this duplicate description.

Reply:
I don't see a duplication.

Reply:
I just saw that I had conflicts in the file.  I took care of them.